### PR TITLE
Upd tg-bot

### PR DESCRIPTION
**Pull request by antonKoiko**
added functions:
1. connecting to the database 
2. checking for news in the database
3. in the absence of news, adding it to the database and sending it to the tg bot

### CODE REVIEW

File: tg_fm_news.py [hunk 1]
```diff
@@ -5,15 +5,25 @@
...
+import datetime
 from bs4 import BeautifulSoup
 from tg import BOT_TOKEN, chat_id
 
 should_exit = False  # Флаг для завершения
 bot = telebot.TeleBot(BOT_TOKEN)
 
+try:
+    #Connecting to the database
+    conn = psycopg2.connect(dbname='NewsDatabase', user='postgres', password='admin', host='localhost')
```
Comment (eqld) [inline tg_fm_news.py]: Read credentials from config file.

File: tg_fm_news.py [hunk 2]
```diff
@@ -5,15 +5,25 @@
...
+except:
+    # if the connection fails, a message will be displayed in STDOUT
+    print('Can`t establish connection to database')
```
Comment (eqld) [inline tg_fm_news.py:21]: Terminate application with non-zero exit code in case pf connection failure.

File: tg_fm_news.py [hunk 3]
```diff
@@ -26,8 +36,22 @@ def print_current_news(interval):
...
+            res = cursor.fetchone()
+
+            if res:
+                print(f"В наличии в БД. Время обновления: {datetime.datetime.now()} ")
+            else:
+                print(f"НЕ в наличии,сейчас добавим и отправим в ТГ-бот.  {datetime.datetime.now()} ")
+                news = (item_text,
+                        item_href,
+                        datetime.datetime.now())
+                cursor.execute("INSERT INTO news (heading, url, date) VALUES (%s, %s, %s)", news)
```
Comment (eqld) [inline tg_fm_news.py:51]: Save to the database AFTER sending to tg.

File: tg_fm_news.py [hunk 4]
```diff
@@ -26,8 +36,22 @@ def print_current_news(interval):
             item_text = item.text
             item_href = "https://sputnik.by/geoBelarus" + item.get("href")
             all_news_dict[item_text] = item_href  # Запись значений в словарь
-            bot.send_message(chat_id, f"{item_text}: {item_href}")
-        # bot.send_message(chat_id, f"{item_text}: {item_href}")
+            # print(f"{item_text}: {item_href}")
+            data = item_text
+            cursor.execute("SELECT * FROM news WHERE heading = %s", (data,))
```
Comment (eqld) [inline tg_fm_news.py]: * Use URL instead of heading as unique key
* Use 'SELECT EXISTS ...' query

File: tg_fm_news.py (hunk 2 above)
Comment (antonKoiko) [reply tg_fm_news.py:21]: fixed

File: tg_fm_news.py (hunk 3 above)
Comment (antonKoiko) [reply tg_fm_news.py:51]: Fixed

File: tg_fm_news.py (hunk 4 above)
Comment (antonKoiko) [reply tg_fm_news.py]: Use URL instead of heading as unique key - fixed

File: tg_fm_news.py (hunk 2 above)
Comment (antonKoiko) [reply tg_fm_news.py:21]: Fixed

File: tg_fm_news.py (hunk 1 above)
Comment (antonKoiko) [reply tg_fm_news.py]: Fixed

File: tg_fm_news.py (hunk 2 above)
Comment (eqld) [reply tg_fm_news.py:21]: With NON-zero exit code.

**Review by eqld — APPROVED** [review]

**Pull request merged**